src/main/java/com/slack/kaldb/logstore/LogStore.java

```diff
@@ -24,6 +23,7 @@
 
   Path getDirectory();
 
-  Collection<String> activeFiles();
+  IndexCommitRefHolder acquireLatestCommit() throws IOException;
```

Comment:
Not super happy that `LogStore` interface is leaking direct lucene abstractions - but it's tough ! We already have it expose `SearcherManager`